src/Sylius/Bundle/ReviewBundle/composer.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
minimum-stability is missing - See https://github.com/Sylius/SyliusProductBundle/blob/master/composer.json#L46.